### ISSUE TITLE
IFLO direction in the Floodplain Cross Sections 

### ISSUE THREAD

**Issue by rpachaly** [FLO-2DSoftware/qgis-flo-2d-plugin#1529]
setting the IFLO direction in the Floodplain Cross Sections can be difficult to understand. We could add a small sketch showing flow directions in a typical cell. Just thinking out loud, but this is something we should consider improving to make it clearer

**Comment by rpachaly** [comment]
@FLO-2DNoemi I added a tooltip into the Plugin following the DIM. See:

![image](https://github.com/user-attachments/assets/75a67c17-ba68-4334-b0f4-977edcde4f1d)

I tried to add an image to the tooltip, but it did not work. We can add this to the documentation and a brief explanation, what do you think @FLO-2DKaren?

![image](https://github.com/user-attachments/assets/697c672f-9e38-4b47-a5b9-be1065e62f34)

**Comment by rpachaly** [comment]
The tooltip is done and merged on https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/pull/1530/files

**Comment by FLO-2DKaren** [comment]
Let's simplify the message.  The negative values are rarely used.  
This will make the box smaller.

<html><head/><body><p>Defines the general direction expected to cross the line. </p><p>IFLO is set to one of the following:</p><p>1 flow to the north</p><p>2 flow to the east</p><p>3 flow to the south</p><p>4 flow to the west</p><p>5 flow to the northeast</p><p>6 flow to the southeast</p><p>7 flow to the southwest</p><p>8 flow to the northwest</p></body></html>

**Comment by FLO-2DKaren** [comment]
![image](https://github.com/user-attachments/assets/a4620a2e-37f8-4622-9ee4-8b970795d949)

**Comment by FLO-2DKaren** [comment]
I just noticed something to improve this system.

- [ ] The trigger is advancing to the next cross section when the direction is assigned.  That is kind of confusing because you expect to see the edit you just made and instead you see the next cross section and it's different.  Remove the advance trigger.  Let the user advance the cross section themselves.
- [ ] Cross sections can be assigned any direction but only two are valid.  All other directions will result in a bad reported data.  It will be invalid.  Let's limit the flow directions based on the angle of the line.
- [ ] When Schema button is ran, we need to close and save the edits instead of showing python error. 

https://github.com/user-attachments/assets/62e361c8-ab9d-46c6-8a36-3c2b27c65214

**Comment by FLO-2DKaren** [comment]
@rpachaly 
Hi Robson,
Before I close this. Can you please look into that trigger issue?

**Comment by rpachaly** [comment]
@FLO-2DKaren 

This is done, check the video: https://flo-2d.sharefile.com/d-sb835fc574aed4191ad55c63a3cf0876e

I did not merged it but created the pull request. On https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/pull/1619

**Comment by FLO-2DKaren** [comment]
Slight issue.  The arrow is the flow direction.  The direction of the water that crosses the floodplain cross sections.  So it should be the opposite of what you did.  You made it the direction of the extension of the cross sections and that is wrong.

Video 18:00

**Comment by FLO-2DKaren** [comment]
All other tests are passing.

**Comment by rpachaly** [comment]
Yeah! You are totally right hahaha. I was so focused on getting the direction of the fpxsec that I forgot it is the flow passing through the fpxsec.  I'll fix it.

**Comment by rpachaly** [comment]
Done and merged on https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/pull/1629
